timer-selector.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8">
     <meta name="description" content="SCP Wiki Timer Selector">
     <meta name="keywords" content="scp,wiki,timer,selector,generator">
-    <link rel="stylesheet" href="/timer/timer-selector.css">
-    <script src="/timer/timer-selector.js"></script>
+    <link rel="stylesheet" href="timer-selector.css">
+    <script src="timer-selector.js"></script>
```

Comment:
You're sure this works when deployed to GitHub Pages?

Reply:
I haven't tried it, but I'd be shocked if it didn't. They're all deployed to the same directory so relative paths should be fine.

Reply:
Can you double check? I'm pretty sure I would've used a non-absolute path back when I set this up if it did work.